src/base/bittorrent/session.cpp

```diff
@@ -127,6 +127,49 @@ namespace
         return result;
     }
 
+    QStringList setToList(const QSet<QString> &set)
```

Comment:
@tgregerson, I hope I don't need to comment all points with same issues. If I comment something you should deal with other similar code too.